XamlControlsGallery/ControlPages/TabViewPage.xaml

```diff
@@ -0,0 +1,246 @@
+                    </muxcontrols:TabView.TabStripHeader>
+                    <muxcontrols:TabView.TabStripFooter>
+                        <TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="6" Style="{ThemeResource BaseTextBlockStyle}" />
+                    </muxcontrols:TabView.TabStripFooter>
+                </muxcontrols:TabView>
+            </local:ControlExample.Example>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;TabView&gt;
+    &lt;TabView.TabStripHeader&gt;
+        &lt;TextBlock Text="TabStripHeader Content" VerticalAlignment="Center" Margin="8,6" Style="{ThemeResource BaseTextBlockStyle}" /&gt;
+    &lt;/TabView.TabStripHeader&gt;
+    &lt;TabView.TabStripFooter&gt;
+        &lt;TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="6" Style="{ThemeResource BaseTextBlockStyle}" /&gt;
+    &lt;/TabView.TabStripFooter&gt;
+&lt;/TabView&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+
+        <local:ControlExample HeaderText="Tab widths can either be equally sized or sized to the content of the tab">
+            <local:ControlExample.Example>
+                <muxcontrols:TabView x:Name="TabView3" SelectedIndex="0" Margin="-12" IsAddTabButtonVisible="False" MinHeight="475">
+                    <muxcontrols:TabView.TabItems>
+                        <muxcontrols:TabViewItem  Header="Home" IsClosable="False">
+                            <muxcontrols:TabViewItem.IconSource>
+                                <muxcontrols:SymbolIconSource Symbol="Home" />
+                            </muxcontrols:TabViewItem.IconSource>
+                            <samplepages:SamplePage1 />
+                        </muxcontrols:TabViewItem>
+                        <muxcontrols:TabViewItem  Header="Tab 2 Has Longer Text" IsClosable="False">
+                            <muxcontrols:TabViewItem.IconSource>
+                                <muxcontrols:SymbolIconSource Symbol="MusicInfo" />
+                            </muxcontrols:TabViewItem.IconSource>
+                            <samplepages:SamplePage2 />
+                        </muxcontrols:TabViewItem>
+                        <muxcontrols:TabViewItem  Header="Third Tab" IsClosable="False">
+                            <muxcontrols:TabViewItem.IconSource>
+                                <muxcontrols:SymbolIconSource Symbol="Placeholder" />
+                            </muxcontrols:TabViewItem.IconSource>
+                            <samplepages:SamplePage3 />
+                        </muxcontrols:TabViewItem>
+                    </muxcontrols:TabView.TabItems>
+                </muxcontrols:TabView>
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
```

Comment:
Here's what it looks like without the negative margins: 

![image](https://user-images.githubusercontent.com/25991996/65171022-cd40b000-d9fe-11e9-9818-4332cbfa76ea.png)

As @chingucoding mentioned, the negative margins is used to overwrite the padding the default example template. I think getting rid of the margins helps reinforce the feeling that the TabView is supposed to take up the whole page. 

Here's another example where I think the negative margin helps:
With:
![image](https://user-images.githubusercontent.com/25991996/65171333-712a5b80-d9ff-11e9-8876-56da64fb1d16.png)

Without:
![image](https://user-images.githubusercontent.com/25991996/65171237-40e2bd00-d9ff-11e9-8d55-96b9c28a22b9.png)

That said, there is still a little bit of space on the right that I cannot account for. I wonder where that is coming from...

Reply:
How do you feel about drawing a box around the entire TabView in each sample? I understand why you want TabView to fill the space, but the visual alignment between TabView and the options panel honestly looks better in your screenshot above compared to my screenshot.
Because TabViewItem content is indented, having multiple layers of margins looks strange. Like NavigationView, TabView wasn't designed to be shoved into a small box inside the page. I'm wondering, but definitely not insisting, whether its own box inside ControlExample would help. Or maybe changing the margins of the content pages to be flush with left/right edges of the tab bar is a better approach.

Reply:
> How do you feel about drawing a box around the entire TabView in each sample?

Would this result in a "box within a box" feeling? Not super keen on that, but I can see how it'd look. I'm worried we'd still have multiple margins *and* more lines.

> TabView and the options panel honestly looks better in your screenshot above compared to my screenshot.

What if I kept the left/top negative margins and got rid of the one on the right? 

> Or maybe changing the margins of the content pages to be flush with left/right edges of the tab bar is a better approach.

To be clear, I think this would require adding negative margins to the sample pages that are referenced in the sample and not remove the margins from the pages themselves. If we removed them from the pages, we'd end up with weirdness in other places, like NavView (where the blue box would touch the NavView):

![image](https://user-images.githubusercontent.com/25991996/65198595-81126180-da38-11e9-9b36-701a0f8f5559.png)

Reply:
Yeah, I'm not sure what to do here. Probably leave the negative margins as they are. This question definitely should not block the PR.